JavaScript/JavaScriptSDK.Tests/CheckinTests/Logging.tests.ts

```diff
@@ -165,7 +166,36 @@ class LoggingTests extends TestClass {
                     Assert.equal("AI (Internal): " + "NONUSRACT_BrowserCannotReadLocalStorage message:\"error!\"", InternalLogging.queue[0].message);
 
                     // cleanup
-                    
+                    InternalLogging.verboseLogging = () => false;
```

Comment:
Would it be safer to read out the original value before setting it to true and then set it back to the original value here rather than explicitly false?